docs/content/topics/developers.md

```diff
@@ -262,6 +262,18 @@ $ helm install -n brigade brigade/brigade -f myvalues.yaml
 
 Don't forget to also create a project.  Check out [projects](./projects.md) to see how it's done.
 
+## Developing brigade with kind
+
+You can also use [kind](https://github.com/kubernetes-sigs/kind) for your day to day Brigade development workflow.
```

Comment:
Could you please also point to the Kind quickstart here?
Thanks!

Reply:
done, thanks!